guides/Guide to Map Folders.md

```diff
@@ -0,0 +1,68 @@
+As of 8/25/2020, the map folders have been reworked to allow expedition areas and POIs to be shared between maps. The major improvement is that in new mapping projects, as long as the mapsize is the same (140x140), you are able to load POIs from Tether on another map.
```

Comment:
I feel like this should go in /maps and/or on the github wiki

Reply:
EEeeeeh, this isn't really applicable to our codebase.

Reply:
Not terribly, no.